backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -66,7 +66,7 @@ export class AdminResolver {
     return await getUserCreations(user.id)
   }
 
-  // @Authorized([RIGHTS.SEARCH_USERS])
+  @Authorized([RIGHTS.SEARCH_USERS])
```

Comment:
This seems to be the wrong right - every function gets its own right.

Reply:
OK, I was not sure while implementing it, I understood that SEARCH_USERS is the actual Admin role / right